src/NuGet.Services.SearchService/Controllers/SearchController.cs

```diff
@@ -197,7 +197,7 @@ private async Task<IHttpActionResult> ExecuteSearchAsync<TResponse>(Func<Task<TR
         {
             try
             {
-                return Json(await searchAction());
+                return Json(await searchAction(), Configuration.Formatters.JsonFormatter.SerializerSettings);
```

Comment:
All `Json` calls will need to pass these `SerializerSettings`? 😞

Reply:
https://github.com/aspnet/AspNetWebStack/blob/ebef5b7d821b64ed5c48765d0caf6ce8a9bcfaf5/src/System.Web.Http/ApiController.cs#L441-L444